### PR TITLE
GSL: bump version

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: sci-libs/gsl/gsl-2.4.recipe [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
```
Comment (fbrosson) [inline sci-libs/gsl/gsl-2.4.recipe]:
```diff
 	cmd:automake
+	cmd:find
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
-	cmd:find
```

**Comment by extrowerk** [comment]
It was a big version bump, but i'm not sure if it is compatible with 1.x or not. It would be maybe better to rename the old recipe to gsl1.
Have you got any info about the compatibility, @korli ?

**Comment by waddlesplash** [comment]
It appears "xaos" and "krita" are the only things that require GSL, so if they still build under this version, then we can merge it.

**Comment by extrowerk** [comment]
@waddlesplash : Xaos compiles fine

**Pull request merged**